pkgs/development/libraries/termcolor/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "termcolor";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "ikalnytskyi";
+    repo = "termcolor";
+    rev = "v${version}";
+    sha256 = "sha256-W0hB+lFJ2sm7DsbOzITOtjJuntSM55BfwUunOOS4RcA=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  cmakeFlags = [ "-DTERMCOLOR_TESTS=ON" ];
+
+  doCheck = true;
+
+  checkPhase = ''
+    ./test_termcolor
+  '';
```

Comment:
```suggestion
  checkPhase = ''
    runHook preCheck
    ./test_termcolor
    runHook postCheck
  '';
```

Reply:
Fixed in 86d00441b0e59e9d915b4ffd675e2a68642aa3da